action.yaml

```diff
@@ -77,7 +77,7 @@ runs:
       id: find-lvh-cli
       shell: bash
       run: |
-        if [ -f "/bin/lvh" ]; then
+        if command -v lvh 2>&1 >/dev/null; then
```

Comment:
This change looks good to me, but do we need all the other changes?

Reply:
Well, it just pure refactoring to improve readability and GH action syntax.
But I can revert them if it doesn't make sense.

Reply:
Actually both (string and number) should work fine.
But the `default` metadata field has type string: https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault.